### ISSUE TITLE
Admissions Sales Associate

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#2188]
Admissions Sales Associate<br>
http://ift.tt/2phIknQ<br><p><img alt="Logo.gif?ixlib=rails-2.1" src="http://ift.tt/2oGPIFg"></p>
<p><strong>Headquarters:</strong> Chicago, IL<br><strong>URL:</strong> <a href="http://ift.tt/1X5Zr70">http://ift.tt/1X5Zr70</a></p>
<div><strong>Our Company</strong></div>
<div><br></div>
<div>Remote Year brings together communities of interesting professionals, entrepreneurs, and freelancers from across the globe to spend a year working, traveling, and exploring 12 cities around the world. Spending one month in each location, communities connect with local cultures and business ecosystems, forming lifelong, borderless personal and professional relationships along the way. Together we are shaping the future of work.</div>
<div><br></div>
<div><strong>Responsibilities</strong></div>
<div><br></div>
<ol>
<li>You will act as a support role to our Admissions Consultants to assist in the growth of our company. Your main responsibilities will include but are not limited to a) moving applicants through the early stages of the application process and b) qualifying applicants<br><br>
</li>
<li>You will nurture interested applicants through high-volume outreach campaigns including phone calls, emails, and text messaging. This will involve inspiring potential Remotes with the Remote Year story and&nbsp;overcoming&nbsp;their objections with the aim of moving qualified applicants forward in the application process, ultimately booking an interview.<br><br>
</li>
<li>You will have full ownership over maintaining our inbound applicant inquiries, utilizing the messages&nbsp;that &nbsp;come&nbsp;in from our website and email campaigns as your largest base of potential applicants while providing excellent customer service.<br><br>
</li>
<li>You will help design and execute systems to better qualify our applicants before they speak with Admissions Consultants<br><br>
</li>
<li>We are a data-driven organization. You will adapt to and utilize our current tracking systems, but will also have the confidence to use your voice to drive improvements in processes and priorities.<br><br>
</li>
</ol><div><strong>Must-have qualifications</strong></div>
<div><br></div>
<ul>
<li>1-2 years of professional experience<br><br>
</li>
<li>Highly motivated by and interested in a career in sales<br><br>
</li>
<li>Team Player<br><br>
</li>
<li>Ability to meet and exceed goals<br><br>
</li>
<li>Strong communication skills<br><br>
</li>
<li>You must be comfortable with change. We are a startup and are constantly evolving<br><br>
</li>
<li>Hunger to learn. Great isn&rsquo;t good enough. You&rsquo;re always looking for ways to improve.<br><br>
</li>
<li>Relationship management skills and openness to feedback.<br><br>
</li>
<li>A "nothing is impossible" attitude!<br><br>
</li>
<li>Experience traveling (preferably internationally)<br><br>
</li>
</ul><div><strong>Nice-to-have qualifications</strong></div>
<div><br></div>
<ul>
<li>Start-up experience<br><br>
</li>
<li>BS/BA degree or equivalent<br><br>
</li>
</ul><div><strong>Qualities</strong></div>
<div><br></div>
<ul>
<li>Passion for the cause &ndash; we need to get every potential Remote excited about the unique way that we are changing the world and you are the face of this<br><br>
</li>
<li>Passion for travel &ndash; role will require physical presence on a Remote Year program (contingent on performance)<br><br>
</li>
<li>Ownership &ndash; we don&rsquo;t like micromanaging; we expect dedication and ownership<br><br>
</li>
<li>Organization skills &ndash; great time management skills, ability to multitask<br><br>
</li>
<li>Interpersonal skills - this role is responsible for quickly conveying the Remote Year brand and experience to potential Remotes<br>
</li>
</ul><div><br></div>
<p><strong>To apply:</strong> Apply online at <a href="http://ift.tt/2oGODND">http://ift.tt/2oGODND</a></p>
<br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
April 25, 2017 at 10:31AM
